引用@tarojs/service@3.0.3 typescript 编译报错 node_modules/@tarojs/service/types/index.d.ts:5:61 - error TS2307: Cannot find module '../src/utils/types'.

<!-- 请不要删除自动生成的 Issue 标签 -->
<!-- 请不要删除自动生成的 Issue 标签 -->
### 相关平台
微信小程序


**小程序基础库: 2.11.3**
**使用框架: React**

### 复现步骤
引用@tarojs/service包，tsc编译
报错 `node_modules/@tarojs/service/types/index.d.ts:5:61 - error TS2307: Cannot find module '../src/utils/types'.`

分析：文件和目录问题

<img src='https://tva1.sinaimg.cn/large/007S8ZIlgy1ggrhkhurasj328k0tmwno.jpg' width='60%' >

### 期望结果
执行编译不报错

### 实际结果
编译报错

### 环境信息
```
Taro CLI 3.0.2 environment info:
    System:
      OS: macOS 10.15.5
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.4.0 - /usr/local/bin/node
      Yarn: 1.22.4 - ~/.yarn/bin/yarn
      npm: 6.14.5 - /usr/local/bin/npm
    npmPackages:
      @tarojs/service: 3.0.3 => 3.0.3
    npmGlobalPackages:
      typescript: 3.6.5 => 3.6.5
```

<!-- generated by taro-issues. 请勿修改或删除此行注释 --><!--labels=T-weapp,V-3,F-react-->

--- a/packages/taro-service/package.json
+++ b/packages/taro-service/package.json
@@ -20,7 +20,8 @@
   "files": [
     "index.js",
     "dist",
-    "types"
+    "types",
+    "src"
   ],
   "keywords": [
     "taro"

